### ISSUE TITLE
[BUG] (OUI Docs) /latest/ re-route does not work 

### ISSUE THREAD

**Issue by KrooshalUX** [opensearch-project/oui#937]
**Describe the bug**
In order for UX to provide accurate annotations to engineering within GitHub and other communications, it is best to utilize /lastest/ URL when pointing to documentatoin on oui.opensearch.com 

However, when visiting the latest documentation version of oui.opensearch.com, /latest/ no longer appears active (can only use 1.0, 1.1, 1.2) 

**To Reproduce**
1. Go to oui.opensearch.org
2. Replace 1.2 with latest
3. See error

**Expected behavior**

/latest/ resolves

**Screenshots**

<img width="907" alt="Screen Shot 2023-07-31 at 10 08 53 AM" src="https://github.com/opensearch-project/oui/assets/105819041/3ebeb48d-0acf-43c9-a37d-82ca976f2fc9">

**Additional context**

This is a critical use case for UX documentation.

**Comment by BSFishy** [comment]
Alternative solution: remove the `1.2` entirely to get latest version.

Example:
[`https://oui.opensearch.org/1.2/#/layout/bottom-bar`](https://oui.opensearch.org/1.2/#/layout/bottom-bar) -> [`https://oui.opensearch.org/#/layout/bottom-bar`](https://oui.opensearch.org/#/layout/bottom-bar)

Without the version, it automatically redirects to the latest version

**Comment by KrooshalUX** [comment]
@BSFishy yes that is a manual edit the UX team can make when handing off. 

We are hoping to bring back the paradigm that was working on the staging site  (though I could have sworn it worked in prod at some point) similar to the OpenSearch docs site https://opensearch.org/docs/latest/dashboards/quickstart/ 

Alt-alt- @kgcreative  had suggested a link at the top of each page "share this" which would automagically remove the version information from the URL.

**Comment by anirudha** [comment]
Closing — this issue has been open for 1-2+ years without progress. If still relevant, please reopen or create a new issue with updated context.
